linode/manager
# feat: [M3-6948] - Add basic routing for VPC

## Description 📝
 Added initial routes, folders, and pages for VPC. Initial routes include:
 - VPC landing:` /vpc`
 - VPC Create: `/vpc/create`
 - VPC details: `/vpc/:id/:tab`

**Worth merging this with PR#9467?** seems like there's some files that overlap
 
## Preview 📷
landing page:
![image](https://github.com/linode/manager/assets/139280159/a3d2da57-f6ba-4b1e-9f57-4c7d757cd79c)

create page:
![image](https://github.com/linode/manager/assets/139280159/7cc11b76-7042-412d-b867-b36cef734041)

details page:
![image](https://github.com/linode/manager/assets/139280159/11a3a492-9402-4d2c-b421-a8278f4175c2)


## How to test 🧪
Make sure the file structure makes sense + checkout this branch and navigate to the urls:

vpc landing:
- http://localhost:3000/vpc

vpc create:
- http://localhost:3000/vpc/create (can also click the create vpc button on the landing page to be brought here)

vpc details:
- http://localhost:3000/vpc/someId/sometab (someId and sometab are just examples, you can put any value for someId / sometab)
